src/modules/storage/BlockStorage.ts

```diff
@@ -63,30 +65,44 @@ export class BlockStorage extends Storages
                 (data.header.validators == undefined) ||
                 (data.header.validators._storage == undefined) ||
                 (data.header.merkle_root == undefined) ||
-                (data.header.signature == undefined)
+                (data.header.signature == undefined) ||
+                (data.txs == undefined)
         ) {
             if (callback != undefined)
                 callback("Parameter validation failed.");
             return;
         }
 
+        var tx_count: number = 0;
+        var enrollment_count: number = 0;
+
+        tx_count = data.txs.length;
+
+        if (data.enrollments)
+            enrollment_count = data.enrollments.length;
+
         var sql =
-        `INSERT INTO blocks
-            (height, prev_block, validators, merkle_root, signature)
-        VALUES
-            (?, ?, ?, ?, ?)
-        ON CONFLICT(height) DO UPDATE SET
-            prev_block = prev_block,
-            validators = validators,
-            merkle_root = merkle_root,
-            signature = signature`;
+            `INSERT INTO blocks
+                (height, prev_block, validators, merkle_root, signature, tx_count, enrollment_count)
+            VALUES
+                (?, ?, ?, ?, ?, ?, ?)
+            ON CONFLICT(height) DO UPDATE SET
+                prev_block = prev_block,
+                validators = validators,
+                merkle_root = merkle_root,
+                signature = signature,
+                tx_count = tx_count,
+                enrollment_count = enrollment_count
+            `;
```

Comment:
There should **never** be a conflict. I'm not sure silent override is a good idea.

Reply:
Of course, INSERT should only happen once. 
BUT, There is currently no detailed error process.
If there is no conflict, it makes the current development and testing process difficult.
This can be eliminated if the error is subdivided.

Reply:
My comment means: Better to show the error than to hide it. It is okay not to handle the error now. But if the error happens, you want to know it. If we know it, we can fix it easily. If we hide it like this, we might loose time looking for it.

Reply:
If so, I will remove the `conflict` part.